Sources/StreamChatUI/Appearance+Images.swift

```diff
@@ -12,21 +12,21 @@ public extension Appearance {
         public var loadingIndicator: UIImage = UIImage(named: "loading_indicator", in: .streamChatUI)!
         public var close: UIImage = UIImage(named: "close", in: .streamChatUI)!
         public var closeCircleTransparent: UIImage = UIImage(named: "close_circle_transparent", in: .streamChatUI)!
-        public var closeCircleFilled: UIImage = UIImage(named: "close_circle_filled", in: .streamChatUI)!
+        public var discardAttachment: UIImage = UIImage(named: "close_circle_filled", in: .streamChatUI)!
```

Comment:
Yep, I like the context-based naming better 👍

Reply:
Done ✅